pkg/volume/glusterfs/glusterfs.go

```diff
@@ -309,7 +309,7 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 
 	p := path.Join(b.glusterfs.plugin.host.GetPluginDir(glusterfsPluginName), b.glusterfs.volName)
 	if err := os.MkdirAll(p, 0750); err != nil {
-		return fmt.Errorf("glusterfs: mkdir failed: %v", err)
```

Comment:
@jsafrane Sure.. I think `Error creating..." is better, I will change to that format.